ibm/resource_ibm_iam_access_group_members.go

```diff
@@ -314,7 +324,12 @@ func resourceIBMIAMAccessGroupMembersDelete(context context.Context, d *schema.R
 		if err != nil {
 			return diag.FromErr(err)
 		}
-		err = iamuumClient.AccessGroupMember().Remove(grpID, *serviceID.IamID)
+
+		removeMembersFromAccessGroupOptions := &iamaccessgroupsv2.RemoveMemberFromAccessGroupOptions{
+			AccessGroupID: &grpID,
+			IamID:         serviceID.IamID,
+		}
+		_, err = iamAccessGroupsClient.RemoveMemberFromAccessGroup(removeMembersFromAccessGroupOptions)
 		if err != nil {
 			return diag.FromErr(err)
```

Comment:
return diag.FromErr(err) ===> return diag.FromErr(fmt.Errorf("Error removing iam ID members to group(%s). API responce: %s", grpID, detailResponse))